relay/messages/execution.go

```diff
@@ -56,16 +57,19 @@ type ExecutionResponse struct {
 }
 
 // ToCircuitRequest converts an ExecutionRequest into a circuit.api.ExecRequest
-func (er *ExecutionRequest) ToCircuitRequest(bundle *config.Bundle, relayConfig *config.Config, useDynamicConfig bool) (*api.ExecRequest, bool) {
+func (er *ExecutionRequest) ToCircuitRequest(bundle *config.Bundle, relayConfig *config.Config, useDynamicConfig bool) (*api.ExecRequest, bool, error) {
 	retval := &api.ExecRequest{}
 	hasDynamicConfig := er.compileEnvironment(retval, relayConfig, useDynamicConfig)
-	executable := bundle.Commands[er.commandName].Executable
-	retval.SetExecutable(executable)
+	command := bundle.Commands[er.commandName]
+	if command == nil {
+		return nil, false, errors.New("Command not found")
```

Comment:
Can move `errors.New("Command not found")` to a file-scoped variable and avoid allocating each time we need it.